### PR TITLE
SV-331 Add new endpoint to enable fetching standard using StandardUId

### PR DESCRIPTION
**Pull request by DevenKShah**

### CODE REVIEW

File: src/SFA.DAS.Courses.Api.UnitTests/Controllers/Standards/WhenCallingGetStandard.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 namespace SFA.DAS.Courses.Api.UnitTests.Controllers.Standards
 {
-    public class WhenCallingGetStandard
+    public partial class WhenCallingGetStandard
```
Comment (CodeWolf86) [inline src/SFA.DAS.Courses.Api.UnitTests/Controllers/Standards/WhenCallingGetStandard.cs:18]: Does this need to be partial anymore?

File: src/SFA.DAS.Courses.Api.UnitTests/Controllers/Standards/WhenCallingGetStandardByStandardUId.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+using SFA.DAS.Courses.Api.Controllers;
+using SFA.DAS.Courses.Application.Courses.Queries.GetStandard;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.Courses.Api.UnitTests.Controllers.Standards
+{
+    public class WhenCallingGetStandardByStandardUId
+    {
+        [Test, MoqAutoData]
+        public async Task Then_Gets_Standard_From_Mediator(
```
Comment (CodeWolf86) [inline src/SFA.DAS.Courses.Api.UnitTests/Controllers/Standards/WhenCallingGetStandardByStandardUId.cs:21]: Gets -> Get ?

File: src/SFA.DAS.Courses.Api.UnitTests/Controllers/Standards/WhenCallingGetStandardByStandardUId.cs [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+            string standardUId,
+            GetStandardByStandardUIdResult queryResult,
+            [Frozen] Mock<IMediator> mockMediator,
+            [Greedy] StandardsController controller)
```
Comment (CodeWolf86) [inline src/SFA.DAS.Courses.Api.UnitTests/Controllers/Standards/WhenCallingGetStandardByStandardUId.cs:25]: Haven't seen `Greedy` used before, I thought that was the default behaviour with frozen standing in for specific items, what significance does it have to this test?

**Review by CodeWolf86 — APPROVED** [review]

File: src/SFA.DAS.Courses.Api.UnitTests/Controllers/Standards/WhenCallingGetStandard.cs (hunk 1 above)
Comment (DevenKShah) [reply src/SFA.DAS.Courses.Api.UnitTests/Controllers/Standards/WhenCallingGetStandard.cs:18]: VS refactoring FAIL

File: src/SFA.DAS.Courses.Api.UnitTests/Controllers/Standards/WhenCallingGetStandardByStandardUId.cs (hunk 3 above)
Comment (DevenKShah) [reply src/SFA.DAS.Courses.Api.UnitTests/Controllers/Standards/WhenCallingGetStandardByStandardUId.cs]: So as I understand Greedy is used to direct autofixture to use the most greedy constructor, in this case there is only one , so perhaps unnecessary, I have removed it.

**Pull request merged**